product_docs/docs/edbcloud/beta/getting_started/01a_managing_quotas.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Managing service limits and constraints in Azure
```

Comment:
Can this be presented as a sub-item under "understanding quotas"? It's not that way on the deployment:

<img width="313" alt="Screen Shot 2021-11-02 at 1 33 43 PM" src="https://user-images.githubusercontent.com/79652654/139947671-f09f536b-fa30-45a7-87cb-373bd7d4633e.png">

Reply:
When checking and remediating using the Azure script, does the user need to know about the content in: https://618272cddb920ad15f2f4e64--edb-docs-staging.netlify.app/docs/edbcloud/latest/getting_started/01_understanding_qotas_in_azure/  

If no, we could use the following structure for the pages and the high-level headings: 

- Preparing your Azure account for BigAnimal (Parent page):
      - Intro
      - Using the Azure Script (with a link to the child page to provide the option 
      of manually checking the configuration) 
      - Remediating the issues
          
          - Manually checking the required configuration (Child page)
                 Understanding the the required configuration (content from above link)
                 Manually checking the configuration using the Azure portal or the Azure CLI

Reply:
I'm not quite following your question, sorry - I think that structure makes sense, but we should expect them to have read the "understanding" page before they run it.

Reply:
You did answer my question.

I have changed the structure. Let me know if it looks OK.